docs/howto/craft-for-platforms/example-python-app.rst

```diff
@@ -0,0 +1,91 @@
+.. _example-python-app:
+
+Example Python app
+==================
+
+This how-to guide covers the steps, decisions, and implementation details that
+are unique when crafting a Python-based snap. We'll work through the aspects
+unique to Python apps by examining an existing recipe.
+
+The process of developing a snap for a Python app builds on top of standard
+Python packaging tools and configuration, making it possible to adapt or
+integrate an app's existing build tooling into the crafting process.
+
+Python projects that follow `PEP 517 <https://peps.python.org/pep-0517/>`_ and
+have a ``pyproject.toml`` must be packed with the ``edge`` channel of
+Snapcraft.
+
+To switch your existing Snapcraft installation to the ``edge`` channel, run:
+
+.. code:: bash
+
+  sudo snap refresh snapcraft --edge
+
+
+To install a parallel copy of Snapcraft with the ``edge`` channel, run:
+
+.. code:: bash
+
+  sudo snap install snapcraft snapcraft_edge --edge
+
+To run the copy, use ``snapcraft_edge`` as the command name.
```

Comment:
> Take note that I consider these to flawed how-tos.

I shouldn't be reviewing the content of the how-tos for incorrect stuff like this, right?

Reply:
If there are technical errors, please point them out!

I'm calling the pages flawed because they're not pure how-tos. The titles aren't exactly appropriate and we're mixing reference material within the text. But given the circumstances they'll serve until we make further strides with the migration.

Reply:
Sounds good, I am going to do another review and read the content in detail.

Reply:
1684dd99af751120f065d1c021d354e9ab201bf4